docs/ru/FMUv6U с поддержкой распределённой сети.md

```diff
@@ -0,0 +1,31 @@
+# Название проекта
+
+[CopterHack-2022](copterhack2022.md), команда **CopterCat**.
+
+## Информация о команде
+
+Состав команды:
+
+(Опишите состав команды: имя и фамилия, контакты (e-mail/имя пользователя в Telegram), роль в команде).
+
+* Лапин Матвей @matveylapin, инеженер/программист.
```

Comment:
`@matveylapin` won't be a link to anything, you may change it to links to `t.me` or something.